Point position
Int64 severity
---
Bool received